### ISSUE TITLE
Loss not converging during training

### ISSUE THREAD

**Issue by Carlogo815** [matterport/Mask_RCNN#1425]
Hi all,

I have been trying to train the newtrok using COCO dataset and the pre-trained coco weights. As far as I understand it, with the default configuration we should be able to obtain a good AP. However, the results I have obtained so far are not that promising. The AP I obtained are lower than 0.01 and the different losses do not converge. Any idea of what could be causing this problem? Any idea or suggestion is appreciated.

Here you can see some of the graphs of the losses that I am obtaining:
![Capture](https://user-images.githubusercontent.com/34351739/55884214-f5062f00-5ba7-11e9-8439-cfdf3871a4fa.PNG)
![Capture2](https://user-images.githubusercontent.com/34351739/55884215-f5062f00-5ba7-11e9-8030-401cbc7ff573.PNG)

**Comment by Marssouss** [comment]
Hi @Carlogo815 , you can see this reply : https://github.com/matterport/Mask_RCNN/issues/527#issuecomment-424632277
I had the same problem, and the point 5 of the reply (that i send you) help me lot :)
